src/testing/Makefile

```diff
@@ -1,17 +1,29 @@
 ## This Makefile is meant to be invoked recursively from the top level directory
 
+## This (SRCS) will include testing_main.cpp
 SRCS	= $(wildcard *.cpp)
-OBJS	= $(SRCS:.cpp=.o)
-DEPS	= $(SRCS:.cpp=.d)
-
 
--include $(DEPS)
+## ** TEMPORARY **
+## the following files aren't working yet; remove them
```

Comment:
So when they are working we will remove them from the undone list so that they will actually run?

Reply:
@kdorheim Right. I have created #450 to track this.